packages/apps/src/initSettings.ts

```diff
@@ -70,10 +75,22 @@ function getApiUrl (): string {
       : 'ws://127.0.0.1:9944'; // nothing found, go local
 }
 
+function migrateStoringAccountStorage () {
+  const localStorageStoragingVersion = store.get('localStorageVersion') as number || 0;
```

Comment:
`storage Storaging` - not sure if that's a typo or not, just FYI :)

Reply:
Good point, done